satpy/readers/virr_l1b.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
```

Comment:
Every python module should copy the header that all other files have. We're still standardizing everything but the copyright author should be "SatPy developers" I think. Blank lines in the header should have `#` also. If you can include a module docstring that would be for the best (describe the file format, what satellites are supported, etc).